common/src/main/java/org/mcaccess/minecraftaccess/Config.java

```diff
@@ -205,20 +205,20 @@ private FallDetector() {}
         public int delay = 2500;
     }
 
-    public static final class ReadCrosshair {
-        private ReadCrosshair() {}
+    public static final class NarrateCrosshair {
+        private NarrateCrosshair() {}
 
         public boolean enabled = true;
         public boolean useJade = true;
-        public boolean speakSide = true;
-        public boolean disableSpeakingConsecutiveBlocks = false;
-        public long repeatSpeakingInterval = 0;
-        public boolean speakAdditionalEntityPoses = true;
+        public boolean narrateBlockFace = true;
+        public boolean disableNarratingConsecutiveBlocks = false;
+        public long narrationRepeatingInterval = 0;
```

Comment:
`narrationRepeatInterval`?

```suggestion
        public long narrationRepeatInterval = 0;
```